### PR TITLE
 Add input check and fix add button styling

### LINKED ISSUE #75: Pressing + button adds empty ingredient button
## Expected Behavior
Pressing + button should not do anything if user hasn't typed in search bar.

## Actual/Current Behavior

User should not be able to press + button and create empty ingredient button.

## Steps to Reproduce the Problem

  1. Run app
  2. Click + button insearch bar.

## Pre-Testing TODOs

Add conditional before allowing creation of ingredient button.

### PR DESCRIPTION
**Pull request by connielion**
## Changes
1. Fixed add button styling.
2. Added check for empty search value.

## Purpose
Fix minor styling issue and add conditional to check for empty input.

## Approach
React typescript and styled components.

Closes #75

### CODE REVIEW

File: src/components/Sidebar.tsx [hunk 1]
```diff
@@ -186,6 +186,7 @@ const Sidebar: FC<SidebarProps> = ({
   };
 
   const renderIngredientsList = () => {
+    console.log(selectedIngredients)
```
Comment (krferrell) [inline src/components/Sidebar.tsx:189]: console.log here

**Review by krferrell — APPROVED** [review]
Looks good other than that console.log

File: src/components/Sidebar.tsx (hunk 1 above)
Comment (Lap343) [inline src/components/Sidebar.tsx:189]: oh no. A console.log

**Review by Lap343 — APPROVED** [review]
Looking good except the console.log

**Review by JorgeLVilla — APPROVED** [review]
Once console.log gets removed should be good to go. Nice job Connie!

**Pull request merged**